EventHandler.java

```diff
@@ -0,0 +1,170 @@
+package SearchDemo;
+
+import com.aliyun.fc.runtime.*;
+import com.aliyun.opensearch.DocumentClient;
+import com.aliyun.opensearch.OpenSearchClient;
+import com.aliyun.opensearch.sdk.dependencies.com.google.common.collect.Maps;
+import com.aliyun.opensearch.sdk.generated.OpenSearch;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchClientException;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchException;
+import com.aliyun.opensearch.sdk.generated.commons.OpenSearchResult;
+import com.aliyun.oss.OSSClient;
+import com.aliyun.oss.model.OSSObject;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+
+import java.io.*;
+import java.util.*;
+
+public class EventHandler implements StreamRequestHandler {
+
+    private static String ossEndpoint = "YourOSSEndpoint";
```

Comment:
Why static here? If this is intended to be constants, then it should add final and name the variable something like OSS_ENDPOINT

Reply:
Good point, will change it to JAVA fashion.